feature/account/setup/src/main/kotlin/app/k9mail/feature/account/setup/navigation/AccountSetupNavigation.kt

```diff
@@ -23,3 +24,16 @@ fun NavGraphBuilder.accountSetupRoute(
         )
     }
 }
+
+fun NavGraphBuilder.nestedAccountSetupRoute(
+    route: String,
+    onBack: () -> Unit,
+    onFinish: (String) -> Unit,
+) {
+    composable(route) {
```

Comment:
I'm not sure I follow. Are you saying it's better for the interface between `:onboarding:main` and `:account:setup` to be `AccountSetupScreen()` rather than `nestedAccountSetupRoute()`? The first one feels like an implementation detail to me. And one that might change in the future (to use navigation instead of manually handling sub screens).

Reply:
My issue is that `nestedAccountSetupRoute()` only existis because of the needs of the onboarding implementation to use a different entry point.